.github/workflows/test.yaml

```diff
@@ -781,7 +629,7 @@ jobs:
           - webpack
           - parcel
           - rollup
-        engine: [binary] #['node-api', 'binary']
+        clientEngine: [node-api] #['node-api', 'binary']
```

Comment:
Why the switch / only testing node-api now?

Reply:
https://github.com/prisma/e2e-tests/pull/1954#discussion_r684442790

Reply:
Changed back to `['binary']`